test/test_nn.py

```diff
@@ -1763,6 +1763,11 @@ def test_batchnorm_eval(self):
             self.assertEqual(res1, res2)
             self.assertEqual(grad1, grad2)
 
+    def test_pairwise_distance(self):
+        input1 = Variable(torch.randn(4, 4), requires_grad=True)
+        input2 = Variable(torch.randn(4, 4), requires_grad=True)
+        gradcheck(lambda x, y: F.pairwise_distance(x, y), (input1, input2))
```

Comment:
So also is needed here?
https://github.com/pytorch/pytorch/blob/master/test/test_nn.py#L675